### ISSUE TITLE
Progress bar with value 0 shows animation at initial render

### ISSUE THREAD

**Issue by Samox** [tamagui/tamagui#2753]
### Current Behavior

I display an empty progress bar (with completion 0) and when it appears on the screen a quick animation is shown from 5% to 0.

### Expected Behavior

I would like the bar to stay empty, and to start filling when the value is not 0

### Tamagui Version

```markdown
1.101.3
```

### Platform (Web, iOS, Android)

```markdown
iOS
```

### Reproduction

```markdown
https://github.com/Samox/tamagui-progress-bar-bug
```

**Comment by Samox** [comment]
https://github.com/tamagui/tamagui/assets/6906559/540f4a9d-be6c-4b67-a137-348a28dce61a

**Comment by natew** [comment]
Fixed in 893b8d810b610adb2793cd879b02e11f9f73fc26
